cloudformation/PolicyTemplates/03-cross-account-policy.yml

```diff
@@ -0,0 +1,24 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Allows users in parsys or infosec prod to call assumeRole for CISDeveloper in infosec-dev."
```

Comment:
How are you allowing parsys users to assume this rule? Are we manually configuring ``CISDeveloper`` group?